charts/camunda-platform/templates/connectors/deployment.yaml

```diff
@@ -0,0 +1,93 @@
+{{- if .Values.connectors.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "connectors.fullname" . }}
+  labels: {{- include "connectors.labels" . | nindent 4 }}
+  annotations: {{- toYaml .Values.global.annotations | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels: {{- include "connectors.matchLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "connectors.labels" . | nindent 8 }}
+        {{- if .Values.connectors.podLabels }}
+        {{- toYaml .Values.connectors.podLabels | nindent 8 }}
+        {{- end }}
+        {{- if .Values.connectors.podAnnotations }}
+        annotations: {{- toYaml  .Values.connectors.podAnnotations | nindent 8 }}
+        {{- end }}
+    spec:
+{{/*      imagePullSecrets:*/}}
+{{/*        {{- include "camundaPlatform.imagePullSecrets" . | nindent 8 }}*/}}
+      containers:
+        - name: connectors
+          image: {{ include "camundaPlatform.imageByParams" (dict "base" .Values.global "overlay" .Values.connectors) }}
+          imagePullPolicy: {{ .Values.global.image.pullPolicy }}
+          {{- if .Values.connectors.containerSecurityContext }}
+          securityContext: {{- toYaml .Values.connectors.containerSecurityContext | nindent 10 }}
+          {{- end }}
+          ports:
+            - containerPort: 8080
```

Comment:
Let's customize this as it's easy and we already do it in some of our charts.

Reply:
I don't feel it makes much sense to do so. To reconfigure it, we have to pass extra variables into spring boot config, which increases complexity, and I don't see benefits of reconfiguring it anyway.
Also, tasklist, operate, and optimize use explicit ports.

Reply:
If it's just Helm config + env var for spring boot, I'd advise doing it.
The current apps are not the stander to follow, we are working to enhance the situation and the new components should be better current ones (especially if it's a small change).

Reply:
Okay, I can do it.

Reply:
Done.